src/radical/pilot/configs/resource_xsede.json

```diff
@@ -357,12 +357,9 @@
         "task_launch_method"          : "SPARK",
         "mpi_launch_method"           : "MPIRUN_RSH",
         "pre_bootstrap_1"             : [ 
-                                        "module purge",
-                                        "module load TACC",
-                                        "module load python/2.7.12",
+                                        "module load python/2.7.9",
                                         "export TACC_DELETE_FILES=TRUE",
```

Comment:
Ah, I see - I had (for some reason) assumed that python versions @TACC are in loockstep...

Thanks, that looks ok then!